packages/rrweb/src/record/iframe-manager.ts

```diff
@@ -8,6 +8,7 @@ import type { StylesheetManager } from './stylesheet-manager';
 
 export class IframeManager {
   private iframes: WeakMap<HTMLIFrameElement, true> = new WeakMap();
+  private attachedIframes: WeakMap<HTMLIFrameElement, true> = new WeakMap();
```

Comment:
I don't really see, what this is doing?
It's being populated in `attachIframe`, yes.
But it's values are never used (or I am not seeing it)

Isn the actual fix just avoiding that an already known iframe gets added again to `iframes` ?

Reply:
If I'm not mistaken we can use a WeakSet instead, since we only care about containment